genie-client/src/main/java/com/netflix/genie/client/exceptions/GenieClientTooManyRequestException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *  Copyright 2022 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.client.exceptions;
+
+/**
+ * An exception class that represents 429 - Too Many Requests received from the server.
+ *
+ * @author andrew
+ * @since 4.0.0
+ */
+public class GenieClientTooManyRequestException extends RuntimeException {
```

Comment:
Should extend `GenieClientException` and be plural `GenieTooManyRequestsException` or `GenieClientTooManyRequestsException` (the latter gets pretty long)

Reply:
Done. Will go with `GenieClientTooManyRequestsException`.